ricecooker/classes/nodes.py

```diff
@@ -212,6 +216,50 @@ def get_json_tree(self):
 
         return tree
 
+
+    def save_channel_children_to_csv(self, metadata_csv, topic_structure = list()):
+        # Not including channel title in topic structure
+        
+        is_channel = isinstance(self, ChannelNode)
+ 
+        if not is_channel:
+            # Build out tag string
+            current_level = self.title
+            # prevent list from being passed by reference. Create new instance of list
+            structure_list = list(topic_structure)
+            # if len(structure_list) < 1:
+            #     structure_list = [current_level]
+            # else:
```

Comment:
It's a good idea to just delete code instead of commenting it out once you know you won't need it. Otherwise, people reading the code may wonder if you intended to add it back later.

Reply:
Sorry about that. Comment has been removed